webhook/server/server.go

```diff
@@ -47,31 +45,24 @@ var (
 
 type WebhookServer struct {
 	context     context.Context
-	cfg         *rest.Config
 	namespace   string
 	webhookType string
 }
 
-func New(ctx context.Context, cfg *rest.Config, namespace, webhookType string) *WebhookServer {
+func New(ctx context.Context, namespace, webhookType string) *WebhookServer {
```

Comment:
NIT: Why not cache `clients` in this `WebhookServer`? Then the below methods do not need input `clients *client.Clients`

Reply:
Updated. Thank you.